content/python/concepts/strings/terms/startswith/startswith.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.startswith()'
+Description: 'Checks whether or not a string starts with a given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.startswith()` method checks a value against a given string and returns `True` if the string starts with that value. Otherwise, it returns `False`.
```

Comment:
```suggestion
The **`.startswith()`** method checks a value against a given string and returns `True` if the string starts with that value. Otherwise, it returns `False`.
```